api/app/signals/apps/services/domain/permissions/signal.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+from django.conf import settings
+from django.db.models import Q
+
+from signals.apps.services.domain.permissions.base import PermissionService as BasePermissionService
+
+
+class PermissionService(BasePermissionService):
+    @staticmethod
+    def _skip_permission_check(permission):
+        """
+        The feature flag can be used to disable the permission, by default it is enabled
+        """
+        flag = f'PERMISSION_{permission.upper()}'
+        if hasattr(settings, 'FEATURE_FLAGS') and flag in settings.FEATURE_FLAGS:
+            return not settings.FEATURE_FLAGS[flag]  # noqa Invert the flag if the check should not be skipped
```

Comment:
We can remove these feature_flags, just was thinking about the ability to enable/disable specific checks especially now that the software is more widely used it can be possible a municipality want's to disable permission via category or department routing.

About the comments and explanations, I just copied what was already implemented. So if this should be changed this is also something that we need to document and explain that it has changed.

Reply:
Renamed the feature flags